src/index.js

```diff
@@ -0,0 +1,98 @@
+import './scss/index.scss';
+import Task from './js/Task.js';
+import List from './js/List.js';
+import { insertTasksIntoDom, taskBlur, taskFocus } from './js/utilities.js';
+import dragDropSorting from './js/sorting.js';
+
+const listElement = document.querySelector('#list');
+const addTaskForm = document.querySelector('#add-task');
+const btnDeleteDoneTasks = document.querySelector('#btn-delete-done-tasks');
+
+const list = new List();
+let { tasks } = list;
+
+const updateIndex = () => {
+  listElement.childNodes.forEach((task) => {
+    dragDropSorting(task, list);
+  });
+};
```

Comment:
It would be a good idea to move this `updateIndex` function to your `utility` folder, to keep your logic separated from your DOM manipulation tasks.

Reply:
I tried moving this method to the `utilities` file, but it was breaking because it needs to have some variables which are available only in the global scope of the index.js file.
1. listElement
2. list - which is the list of all tasks returned from the `List` class.

This is why it breaks when I move it.

Reply:
What if you move them too and export them to your Index file? this way you keep your Index cleaner of global variables and you keep your logic in separated modules.

Reply:
Well, the `list` variable has some value assignments inside the index file, and if I move it so many other things will break.